.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+Thanks for opening an issue for us!  Before you open an issue, please check if a similar issue exists or has been closed before.
+
+If you're asking a question about how to use Blockly in your application, please ask questions on the [mailing list](https://groups.google.com/forum/#!forum/blockly) instead of filing issues.
+
+_Delete the above section and the instructions in the sections below before submitting.  If you don't do this, we'll assume you didn't bother to read the instructions or try to solve the problem before filing an issue.  If a section is not applicable, type "N/A" in that section._
+
+### Problem statement
+
+_Remove this hint: these checkboxes can be checked like this: [x]_
+
+- [ ] Bug report
+- [ ] Feature request
+
+_What were you trying to do when you hit this bug?  Or, what do you want to do with Blockly that your proposed feature would enable?_
+
+### Expected Behavior
+
+_Please describe what should happen.  Include screenshots if applicable._
+
+### Actual Behavior
+
+_Describe what actually happens.  Include screenshots if applicable._
+
+### Steps to Reproduce
+
+_Explain what someone needs to do in order to see what's described in *Actual behavior* above_
+
+### Stack Traces
+
+```
+Please open up the console.  If you see any Blockly-related errors, paste them here.
+
+Ignore any instances of "Uncaught (in promise) DOMException: The play() request was interrupted by a call to pause()."
+```
+
+### Operating System and Browser
+
+_Remove this hint: these checkboxes can be checked like this: [x]_
+
+Desktop:
+- [ ] Chrome
+- [ ] Firefox
+- [ ] Safari
+- [ ] Opera
+- [ ] IE 10+
+- [ ] IE 11
+- [ ] EDGE
+
+- [ ] Smartphone (please complete the following information):
```

Comment:
Smartphone shouldn't be a checkbox if Desktop isn't a checkbox.

Reply:
Desktop is now a checkbox.